Tests/BasicsTests/AsyncProcessTests.swift

```diff
@@ -22,58 +23,65 @@ import class TSCBasic.Thread
 import func TSCBasic.withTemporaryFile
 import func TSCTestSupport.withCustomEnv
 
-final class AsyncProcessTests: XCTestCase {
-    func testBasics() throws {
+@Suite(
+    // because suite is very flaky otherwise. Need to investigate whether the tests can run in parallel
+    .serialized
+)
+struct AsyncProcessTests {
+    @Test
+    func basics() throws {
         do {
             let process = AsyncProcess(args: "echo", "hello")
             try process.launch()
             let result = try process.waitUntilExit()
```

Comment:
This line is presumably actually blocking the thread, which is dangerous in Swift Testing because tests run on the shared thread pool. Consider switching over to an asynchronous suspension.